tests/pipelines/test_pipelines_text2text_generation.py

```diff
@@ -83,6 +83,39 @@ def test_small_model_pt(self):
         ]
         self.assertEqual(outputs, target_outputs)
 
+        import torch
```

Comment:
(nit) for consistency we could add this import to the top with a `is_torch_available()` statement

Reply:
Done !